packages/cashier/src/pages/on-ramp/on-ramp.scss

```diff
@@ -1,4 +1,8 @@
 .on-ramp {
+    @include desktop-screen {
+        padding: 0 0.8rem;
+    }
+
     @include tablet-screen {
         max-width: 60rem;
     }
```

Comment:
alternatively you can also just set this as both tablet and mobile.

Reply:
Can't really. Mobile is 100vw and tablet is 60rem.

Reply:
Oh, actually this is not even required.